AirCastingTests/BluetoothProtectorTests.swift

```diff
@@ -0,0 +1,267 @@
+import XCTest
+import Resolver
+import Foundation
+@testable import AirCasting
+
+class BluetoothProtectorTests: ACTestCase {
+    let databaseSpy = AirBeamDatabaseSpy()
+    lazy var sut = BluetoothConnectionProtector()
+    
+    func testWrongStatuses_whenAskedToPredicate_shouldResultWithZero() throws {
```

Comment:
applies to all tests here:
Try to make names of tests less focused on what  the test is doing and more on what production code is doing and expectations. When naming tests imagine that someone only gets test report and so they only see the failing test name - it would be hard to immediately tell what failed here. Maybe something like:
`test_whenAsksDatabaseForExistingSessions_requestsOnlyMobileSessions` (not sure if that's true here, but hopefully you get the idea)